addons/vela-prism/resources/vela-prism.cue

```diff
@@ -0,0 +1,14 @@
+output: {
+	type: "helm"
+	properties: {
+		repoType: "helm"
+		url:      "https://charts.kubevela.net/prism"
+		chart:    "vela-prism"
```

Comment:
I think the vela-prism will rely-on the fluxcd addon if use this way. I'm not sure if this proper? ping @Somefive

Reply:
Let it rely on fluxcd for now. It's okay to do so.